pygame_gui/ui_manager.py

```diff
@@ -225,6 +226,7 @@ def process_events(self, event: pygame.event.Event):
                             # this is not a mistake.
 
                             break
+            return consumed_event
```

Comment:
Looks like this line needs to go left by four spaces (one level of indentation) otherwise it will quit out early from the to level loop here when you have more than one layer in the UI.